src/styles/index.scss

```diff
@@ -1,3 +1,58 @@
+@import './utils/variables';
+@import 'block/header';
+@import 'block/pruduct-card';
+@import 'block/starts';
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 400;
+  font-style: normal;
+}
+
+.main {
+  display: grid;
+  grid-template-columns: repeat(4, 200px);
+  gap: 46px 48px;
+  justify-content: center;
+  padding: 50px 40px;
+}
+
+@media (max-width: 487px) {
+  .main {
+    grid-template-columns: repeat(1, 200px);
+  }
+}
+
+@media (min-width: 488px) {
+  .main {
+    grid-template-columns: repeat(2, 200px);
+  }
+}
+
+@media (min-width: 768px) {
+  .main {
+    grid-template-columns: repeat(3, 200px);
+  }
+}
+
+@media (min-width: 1024px) {
+  .main {
+    grid-template-columns: repeat(4, 200px);
+  }
+}
+
+.reviews {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 16px;
+}
+
+.reviews__review {
+  display: flex;
+  align-items: center;
+  margin: 0;
+  font-size: 10px;
+  color: $primary-color;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), variables should be used for font sizes as well. Consider defining a variable for the font size used here.